connect/mirror-client/src/main/java/org/apache/kafka/connect/mirror/RemoteClusterUtils.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.connect.mirror;
+
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.clients.consumer.OffsetAndMetadata;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+import java.time.Duration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+ 
+/** Convenience methods for multi-cluster environments. */
+public final class RemoteClusterUtils {
```

Comment:
The methods here are slightly different from what was described in the KIP but you're using the helper class to create AdminClient so I'm on with it (as long as we update the KIP at some point).

Reply:
I'll update the KIP and call out the changes to the discuss thread prior to marking this ready-for-review.

Reply:
Update: I've broken the missing methods out into TopicFilter, GroupFilter, ConfigPropertyFilter, instead of having them all in ReplicationPolicy. lmk what you think.